### PR TITLE
Update version and website in Manifests and recipes

### LINKED ISSUE #5857: Update module manifest url and maybe version?
As @hishamco pointed out on https://github.com/OrchardCMS/OrchardCore/pull/5815

the manifests for the modules are all set to v2.0.0, and the website also could be updated to `https://orchardcore.net`
```
    Website = "https://orchardproject.net",
    Version = "2.0.0"
```

What version number do we want to release with?

### PR DESCRIPTION
**Pull request by agriffard**
```
    Author = ManifestConstants.OrchardCoreTeam,
    Website = ManifestConstants.OrchardCoreWebsite,
    Version = ManifestConstants.OrchardCoreVersion,
```

Fixes #5857

### CODE REVIEW

**Comment by hishamco** [comment]
I just wanna to push the same PR ;)

**Comment by hishamco** [comment]
I think you miss `OrchardVersion`

**Comment by agriffard** [comment]
> I think you miss `OrchardVersion`

There is no such property in OC. It was in O1.

**Comment by agriffard** [comment]
> Some of the test and template projects have version that differs from 1.0.0-rc2.

I didn't change them on purpose.
Template projects: '0.0.1'

**Comment by hishamco** [comment]
`Version` make me confused, I'm expected a module version not a OC version

File: test/OrchardCore.Tests.Pages/OrchardCore.Modules.Pages/Module.Pages/Manifest.cs [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 [assembly: Module(
     Name = "Module Pages",
-    Author = "The Orchard Team",
-    Website = "https://orchardproject.net",
+    Author = ManifestConstants.OrchardCoreTeam,
+    Website = ManifestConstants.OrchardCoreWebsite,
     Version = "2.0.0",
```
Comment (Skrypt) [inline test/OrchardCore.Tests.Pages/OrchardCore.Modules.Pages/Module.Pages/Manifest.cs:7]: Why are we keeping 2.0.0 here?

**Pull request merged**